kotlin-reference-server/src/main/kotlin/org/stellar/reference/plugins/ConfigureRouting.kt

```diff
@@ -53,4 +54,7 @@ fun Application.configureRouting(cfg: Config) = routing {
   if (cfg.sep24.enableTest) {
     testSep24(helper, depositService, withdrawalService, cfg.sep24.interactiveJwtKey)
   }
+  if (cfg.appSettings.isTest) {
```

Comment:
Do we need to keep two settings for `sep24` and `appSettings`. They seems to serve the same purpose.

Reply:
Hm...not really? I will merge them.

Reply:
Actually, I think we need to keep them. If we are deploying the reference server, we would want to enable the SEP-24 routes.

Reply:
Hm... I guess the configuration `sep24.enableTest` is confusing from my (user's) perspective.... It actually simulate the anchor's sep-24 flow.